tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -0,0 +1,365 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier4
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import get_all_pvcs, delete_pvcs
+from ocs_ci.ocs.resources.pod import get_all_pods, get_fio_rw_iops
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.resources.pod import (
+    get_mds_pods, get_mon_pods, get_mgr_pods, get_osd_pods
+)
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check, run_cmd
+from tests.helpers import (
+    verify_volume_deleted_in_backend, wait_for_resource_state
+)
+from tests import disruption_helpers
+
+log = logging.getLogger(__name__)
+
+
+class DisruptionBase(ManageTest):
```

Comment:
Removed class DisruptionBase as it seems to be unnecessary because it was used to execute the actual test steps that can be effectively added in actual test method.

Reply:
Thasnks @jilju, could you please move verify_resource_deletion() to a commonplace as it's implemented in the same way in more than 1 place

Reply:
Done